### ISSUE TITLE
Pidgin leaves handle on files after transfers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#37]
```
What steps will reproduce the problem?
1. Send a file with Pidgin.
2. While Pidgin is still open, try deleting that file.
3. Despite the transfer being complete, Pidgin is still "using it".

What is the expected output? What do you see instead?
Just a little annoying bug while I was transferring some files to a friend.
I had all of them on my desktop, and after sending them I couldn't delete
any until I closed Pidgin. Process Explorer told me that Pidgin still had a
"handle" on them (though I couldn't find it in the Pidgin process to delete
manually)

What version of the product are you using? On what operating system?
0.0.11 - XP SP2 - Pidgin 2.4.1


Please provide any additional information below.


```

Original issue reported on code.google.com by `mastaste...@gmail.com` on 11 Apr 2008 at 3:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `felipe.contreras` on 4 May 2008 at 9:53
- Added labels: **Milestone-0.1.0**

**Comment by GoogleCodeExporter** [comment]
```
this is a pidgin issue, already reported, fixed and now broken again
the handles of transfered files are not closed, nothing to do with pecan
```

Original comment by `sher...@gmail.com` on 22 Dec 2008 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
sherpya: great! do you know the bug report id?
```

Original comment by `felipe.contreras` on 22 Dec 2008 at 10:59

**Comment by GoogleCodeExporter** [comment]
```
this is bug #1643 on Pidgin trac: http://developer.pidgin.im/ticket/1643
```

Original comment by `salo.r...@gmail.com` on 29 Jan 2009 at 8:33

**Comment by GoogleCodeExporter** [comment]
```
Pidgin bug #1643 points out to the msn plugin, so in theory msn-pecan has the 
same
problem.

Is there a bug report for libpurple core?
```

Original comment by `felipe.contreras` on 6 Jun 2009 at 1:38

**Comment by GoogleCodeExporter** [comment]
```
Considering this is present on libpurple too I'll aim this to 0.2.0.
```

Original comment by `felipe.contreras` on 11 Jul 2009 at 3:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `felipe.contreras` on 11 Jul 2009 at 3:12
- Added labels: **Milestone-0.2.0**
- Removed labels: **Milestone-0.1.0**

**Comment by GoogleCodeExporter** [comment]
```
I think this might be fixed in 0.1.0. A lot of memory leaks have been removed, 
and 
with that the possibility of leaving the fd open. Please verify (on git or 
0.1.0).
```

Original comment by `felipe.contreras` on 19 Jul 2009 at 12:10
- Changed state: **Fixed**
